bcgov/onroutebc
# Orv2 1122

# Description

This PR defined initial permit payment data implementation, including database tables and entity objects.

## Type of change


# How Has This Been Tested?

Test steps:

1. Run "docker compose up --build -d sql-server-db" to start up a new db docker container.
2. Verify new tables for "ORBC_PAYMENT", "ORBC_TRANSACTION", "ORBC_PAYMENT_TRANSACTIONS" and "ORBC_PAYMENT_PERMIT".
3. Verify backend\vehicles\src\modules\payment\entities\payment.entity.ts and backend\vehicles\src\modules\payment\entities\transaction.entity.ts.



## Checklist






## Further comments
This is a initial version for the data design of the payment.  It needs to be updated after review. Please let me know if any issues, concerns or suggestions in this change. Thanks.


---

Thanks for the PR!

Any successful deployments (not always required) will be available below.
[Backend - Vehicles](https://onroutebc-533-backend-vehicles.apps.silver.devops.gov.bc.ca/) available
[Backend - DMS](https://onroutebc-533-backend-dms.apps.silver.devops.gov.bc.ca/) available
[Frontend](https://onroutebc-533-frontend.apps.silver.devops.gov.bc.ca/) available

Once merged, code will be promoted and handed off to following workflow run.
[Main Merge Workflow](https://github.com/bcgov/onroutebc/actions/workflows/merge-main.yml)
